src/main/java/dev/dokan/dokan_java/examples/DirListingFileSystem.java

```diff
@@ -93,7 +94,7 @@ public int zwCreateFile(WString rawPath, DokanIOSecurityContext securityContext,
             }
         }
 
-        long val = this.handleHandler.incrementAndGet();
+        @Unsigned long val = this.handleHandler.incrementAndGet();
```

Comment:
Here the annotation is actually not needed, since this is "just" an example filesystem implementation.

Reply:
Objection: In my opinion the example filesystem is the best place to teach usage of the annotation.

Reply:
In this case the sole purpose of the defined vars is to be returned to the filesystem. Therefore flagging them as "special" seems appropriate to me. The DirListingFileSystem should teach the correct way of using the dokan-java API and the best way to use it, is to consistently keep track of unsigned values.